lib/index.js

```diff
@@ -37,6 +38,10 @@ module.exports = function Cache(opts) {
     caches.push(new FSStorage({ path: opts.fsCachePath }));
   }
 
+  if (opts.caches) {
```

Comment:
is it useful to have the caches be completely configurable? let the user determine the order of the caches. e.g. maybe I have something that I'd want to put in between the memory & fs caches?  You could accomplish something like that by having a sort of registry of string->cache types.  e.g.
```js
new Cache({
  ...otherOpts,
  caches: [ 'MemoryCache', new FooCache(), new BarCache(), 'FSCache' ]
})
```

Reply:
The string–>object registry pattern is one I've personally had some trouble with in JavaScript. It tends to be easier to let them pass in literal or instantiated objects. `winston@1` suffered from this problem. 

Perhaps a better solution is to have a second option that lets the consumer define everything, vs. inject into the end. e.g.

``` js
// Goes at the end
new Cache({
  ...otherOpts,
  caches: [new OtherCache()] 
});

// Defines everything
const { MemoryCache, FSCache } = require('out-of-band-cache');
new Cache({
  ...otherOpts,
  override: true,
  caches: [new MemoryCache(), new OtherCache(), new FSCache()] 
});
```

Either way I see that as a non-blocking feature addition to this PR.

Reply:
I feel like the wording of the `caches` property is what gets me though.  In my head that implies "here are the caches I want you to use" and not "here's some additional fallback mechanisms"

Reply:
Naming things is hard. What if we made it `fallback`? e.g.:

``` js
// Goes at the end
new Cache({
  ...otherOpts,
  fallback: [new OtherCache()] 
});

// Defines everything
const { MemoryCache, FSCache } = require('out-of-band-cache');
new Cache({
  ...otherOpts,
  caches: [new MemoryCache(), new OtherCache(), new FSCache()] 
});
```

Reply:
I could get behind that

Reply:
Yea `caches` does imply to me a definitive list. Another property seems reasonable